frontend/src/styles/global.scss

```diff
@@ -553,6 +553,7 @@ input::-ms-clear {
     min-height: 40px;
     border: 1px solid var(--border);
     border-radius: var(--radius);
+    background-color: var(--white);
```

Comment:
Somehow this feels wrong. E.g. what if we want dark mode? Could `var(--bg-light)` work better?

Reply:
Yeah --bg-light should be used 👍 

I think this won't help us that much regarding dark mode though 😆